makefile

```diff
@@ -0,0 +1,8 @@
+VERSION ?= 5.2.0
+BRANCH ?= develop
+
+.PHONY: all
+all: tar
+
+tar:
+	tar cfz prodbin-$(BRANCH)-$(VERSION).tar.gz Products bin
```

Comment:
Can you switch it to $(VERSION)-$(BRANCH)  determine branch by running something like git rev-parse --abbrev-ref HEAD